dnf-behave-tests/dnf/dnf-automatic/reboot.feature

```diff
@@ -0,0 +1,101 @@
+@no_installroot
+Feature: dnf-automatic reboots
+
+
+Background:
+Given I delete file "/etc/yum.repos.d/*.repo" with globs
+  And I create file "/etc/dnf/dnf.conf" with
+    """
+    [main]
+    plugins=0
+    """
+
+
+Scenario: dnf-automatic does not reboot when reboot = never
+  Given I create file "/etc/dnf/automatic.conf" with
+    """
+    [commands]
+    reboot = never
+    reboot_command = "echo 'I LOVE REBOOTING'"
+
+    [emitters]
+    emit_via = stdio
+    """
+    And I use repository "simple-base"
+    And I successfully execute dnf with args "install labirinto"
+    And I use repository "simple-updates"
```

Comment:
What about adding this to `Background` and just use the same repo for every scenario? I believe it would make the code more readable and each scenario much shorter. For instance, if you use the repos `dnf-ci-fedora{,updates}` then you can play with packages that do not require reboots as well like `abcde`.

Reply:
How's that, I still kept the test with the kernel upgrade the same but the rest are shorter.